source/common/http/http1/conn_pool.cc

```diff
@@ -220,6 +229,11 @@ void ConnPoolImpl::onResponseComplete(ActiveClient& client) {
 
 void ConnPoolImpl::onUpstreamReady() {
```

Comment:
Is this supposed to be triggered on next poll batch?

Reply:
yes, I checked the logs and that seems to be the case.